typify-impl/src/value.rs

```diff
@@ -773,7 +1025,11 @@ mod tests {
 
         assert_eq!(
             type_entry
-                .output_value(&type_space, &json!(null))
+                .output_value(
+                    &type_space,
+                    &json!(null),
+                    TokenStream::from_str("super::").unwrap()
```

Comment:
for consistency and simplicity, please use
`quote!{ super:: }`
rather than
`TokenStream::from_str("super::").unwrap()`
thoughout